vcpkg/consume/manifest-mode.md

```diff
@@ -0,0 +1,331 @@
+  
+```xml
+<PropertyGroup Label="Vcpkg">
+  <VcpkgEnableManifest>true</VcpkgEnableManifest>
+</PropertyGroup>
+```
+
+Alternatively you can enable manifest mode in your MSBuild call by passing `msbuild /p:VcpkgEnableManifest=true` as a parameter.
+
+```Console
+PS D:\projects\manifest-example> msbuild /p:VcpkgEnableManifest=true
+MSBuild version 17.7.0-preview-23319-02+6829506b8 for .NET Framework
+Build started 8/11/2023 11:29:50 AM.
+
+Project "D:\projects\manifest-example\manifest-example.sln" on node 1 (default targets).
+ValidateSolutionConfiguration:
+  Building solution configuration "Debug|x64".
+Project "D:\projects\manifest-example\manifest-example.sln" (1) is building "D:\projects\manifest-example\manifest-example.vcxproj" (2) on node 1 (default targets).
+PrepareForBuild:
+  (omitted)
+InitializeBuildStatus:
+  (omitted)
+ComputeStdModulesCompileInputs:
+  (omitted)
+SetModuleDependencies:
+  Creating directory "x64\Debug\manifest.ceffc6eb_MD.tlog\".
+VcpkgTripletSelection:
+  Using triplet "x64-windows" from "D:\projects\manifest-example\vcpkg_installed\x64-windows\x64-windows\"
+  Using normalized configuration "Debug"
+VcpkgInstallManifestDependencies:
+  Installing vcpkg dependencies to D:\projects\manifest-example\vcpkg_installed\x64-windows\
+  Creating directory "D:\projects\manifest-example\vcpkg_installed\x64-windows\".
+  "D:\vcpkg\vcpkg.exe" install  --x-wait-for-lock --triplet "x64-windows" --vcpkg-root "D:\vcpkg\" "--x-manifest-root=D:\projects\manifest-example\" "--x-install-root=D:\projects\manifest-example\vcpkg_installed\x64-windows\"
+  "D:\vcpkg\vcpkg.exe" install  --x-wait-for-lock --triplet "x64-windows" --vcpkg-root "D:\vcpkg\" "--x-manifest-root=D:\projects\manifest-example\" "--x-install-root=D:\projects\manifest-example\vcpkg_installed\x64-windows\"
+  Detecting compiler hash for triplet x64-windows...
+  The following packages will be built and installed:
+      cxxopts:x64-windows -> 3.1.1
+      fmt:x64-windows -> 10.0.0
+      range-v3:x64-windows -> 0.12.0#1
+    * vcpkg-cmake:x64-windows -> 2023-05-04
+    * vcpkg-cmake-config:x64-windows -> 2022-02-06#1
+  (omitted)
+ClCompile:
+  (omitted)
+Link:
+  (omitted)
+AppLocalFromInstalled:
+  pwsh.exe -ExecutionPolicy Bypass -noprofile -File "D:\vcpkg\scripts\buildsystems\msbuild\applocal.ps1" "D:\projects\manif
+  est-mode-msbuild\x64\Debug\manifest-example.exe" "D:\projects\manifest-example\vcpkg_installed\x64-windows\x64-windows\debug\bin"
+  "x64\Debug\manifest.ceffc6eb.tlog\manifest-example.write.1u.tlog" "x64\Debug\vcpkg.applocal.log"
+  D:\projects\manifest-example\x64\Debug\fmtd.dll
+FinalizeBuildStatus:
+  (omitted)
+Done Building Project "D:\projects\manifest-example\manifest-example.vcxproj" (default targets).
+
+Done Building Project "D:\projects\manifest-example\manifest-example.sln" (default targets).
+
+Build succeeded.
+```
+
+### [Visual Studio](#tab/build-visual-studio)
```

Comment:
This will need a "Visual Studio (CMake)" tab too, and this one should be named "Visual Studio (MSBuild)".

Reply:
I would suggest adding that in an update to this article